test/integration/vaultauthmethods/terraform/main.tf

```diff
@@ -36,19 +36,18 @@ resource "kubernetes_namespace" "tenant-1" {
   }
 }
 
-resource "kubernetes_service_account" "test-sa" {
+resource "kubernetes_default_service_account" "default-sa" {
```

Comment:
nit: the resource name also includes the resource type, suggest just calling it `default`